alecs-theme/src/WEB-INF/src/resources-importer/journal/articles/Card Row/Shirts.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0"?>
+
+<root available-locales="en_US" default-locale="en_US">
+	<dynamic-element name="HTML18zr" type="text_area" index-type="text" instance-id="zefr">
+		<dynamic-content language-id="en_US"><![CDATA[<div class="card card-type-asset">
+<div class="aspect-ratio card-item-first">
+<div class="aspect-ratio-item-center-middle aspect-ratio-item-fluid card-type-asset-icon" style="     padding: 0;     margin: 0; "><img src="https://cdn.shopify.com/s/files/1/0407/0829/products/White_t-shirt_front_1024x1024.jpg" /></div>
+</div>
+
+<div class="card-body">
+<div class="card-row">
+<div class="autofit-col autofit-col-expand"><a href="//amazon.com" target="_blank"><span class="icon-shopping-cart icon-monospaced pull-right">&nbsp;</span></a>
+
+<div class="card-title text-truncate" title="White Shirt">White Shirt</div>
+
+<div class="card-subtitle text-truncate" title="Size: Medium">Medium</div>
+
+<div class="card-detail"><span class="label label-success"><span class="label-item label-item-expand">$20.00</span> </span></div>
+</div>
+</div>
+</div>
+</div>]]></dynamic-content>
```

Comment:
Instead of all this inline HTML markup, can we put it into the journal's template?  The article should only contain the contents of the data.